### ISSUE TITLE
Canvas Scaler Match value has to be set to the left (width) or tooltips will be offset and flicker.

### ISSUE THREAD

**Issue by James9074** [James9074/Unity-Tooltips#1]
See http://puu.sh/kJG7m/9337af4e3e.png for an example.

**Comment by rmmfree** [comment]
You can fix it replacing all occurrences of Screen.width by mCanvasWidth, in Tooltip.cs file.

**Comment by James9074** [comment]
@rmmfree Woah do you use Unity Tooltips? I didn't realize anyone used this thing! Want me to get it updated for the latest Unity versions? I'll see about replacing Screen.width as well - thanks for the tip.

**Comment by rmmfree** [comment]
Yes, I'm using it since one year or so in professional projects. 
Sorry, but my solution is not working well, I've tested it in other resolutions and it's not correct. Your code is fine. It flickers on mouse in, but only in a certain case. I keep testing it, and if I find the solution I'll post it here.
Thanks for sharing this plugin.

**Comment by James9074** [comment]
@rmmfree That's awesome! Well if you have any feature requests let me know, I'd be happy to take a look. I always wanted to extend this to handle images and more dynamic content in the tooltips.
